Source/Events.Processing/Filters/RemoteFilterDefinition.cs

```diff
@@ -8,23 +8,28 @@ namespace Dolittle.Runtime.Events.Processing.Filters
     /// <summary>
     /// Represents a <see cref="IFilterDefinition" /> for a remote filter.
     /// </summary>
-    public class RemoteFilterDefinition : IFilterDefinition
+    public class RemoteFilterDefinition : IPersistableFilterDefinition
```

Comment:
It looks a bit strange that in the core Runtime code, there is an `IFilterDefinition`, and then two implementations `RemoteFilterDefinition` and `TypedFilterWithEventSourcePartitionDefinition` - while in the MongoDB storage there are just `FilterDefinition` and `TypePartitionFilterDefinition` that correspond to the `RemoteFilterDefinition` and `TypedFilterWithEventSourcePartitionDefinition` respectively.

It seems like these two should represent eachother 1-to-1 ?

Reply:
Yes, it is strange. Registering an issue for it.